core/zts/src/main/rdl/RoleCert.rdli

```diff
@@ -16,3 +16,16 @@ resource RoleCertificate POST "/rolecert" (name=PostRoleCertificateRequestExt) {
         ResourceError UNAUTHORIZED;
     }
 }
+
+// Fetch all roles that are tagged as requiring role certificates for service
+resource RoleList GET "/rolecert?service={service}" (name=getRolesRequireRoleCert) {
```

Comment:
we already use POST /rolecert to request role certs so with the same URI, it's somewhat not clear that we're asking for a list of roles that require role certs. Maybe we separate the uri: GET /role/cert?...

Reply:
Fixed